containers/tfc-toolchain/install-common.sh

```diff
@@ -20,17 +20,15 @@ hide_output apt install -y --no-install-recommends \
   libtool \
   autopoint \
   gperf \
-  python3-distutils \
   python3-jinja2 \
   python3-venv \
-  gcc-13 \
-  g++-13 \
-  g++-13-aarch64-linux-gnu \
+  gcc-14 \
+  g++-14 \
+  g++-14-aarch64-linux-gnu \
   libltdl-dev # Required for autoconf
 
 # Install documentation specific packages
 hide_output apt install -y --no-install-recommends \
-  doxygen \
```

Comment:
Now built from source, the doxygen binary had some strange llvm runtime issues, probably because of two differently build llvm libraries in the container, from apt and our built from source